pyanaconda/modules/localization/localization.py

```diff
@@ -43,6 +46,14 @@ class LocalizationService(KickstartService):
 
     def __init__(self):
         super().__init__()
+        self._available_translations = []
+        for lang in get_available_translations():
+            tdata = LanguageData()
+            tdata.english = get_english_name(lang)
+            tdata.lid = lang
+            tdata.native = get_native_name(lang)
+            self._available_translations.append(tdata)
```

Comment:
It might be good form to make this a private helper, so the init is really just a series of `field=something`.

Reply:
SInce now the two structures are seperate and this is moved to method bodies I feel like private helper is an overkill/